app/controllers/supplementary/SummaryPageController.scala

```diff
@@ -52,7 +52,7 @@ class SummaryPageController @Inject()(
   def displayPage(): Action[AnyContent] = authenticate.async { implicit request =>
     customsCacheService.fetch(supplementaryCacheId).map {
       case Some(cacheMap) => Ok(summary_page(appConfig, SupplementaryDeclarationData(cacheMap)))
-      case None           => Ok(summary_page(appConfig, SupplementaryDeclarationData()))
+      case None           => BadRequest(summary_page_no_data(appConfig))
```

Comment:
I'm not sure about BadRequest here. It's not an error, but an empty summary page, so `summary_page_no_data` should be enough